docs/html/reference/pip_install.rst

```diff
@@ -128,6 +128,7 @@ and like arguments to :ref:`pip install`, the following forms are supported::
     [[--option]...]
     <requirement specifier> [; markers] [[--option]...]
     <archive url/path>
+    <requirement specifier> [; markers] [[--option]...] @ <archive url/path>
```

Comment:
More likely `<requirement specifier> @ <archive url/path> [; markers] [[--option]...] `
But since `Requirement Specifiers` section talks about PEP508 (which defines the `@ <archive url/path>` syntax), I'd say this line is redundant (and should be removed).

Reply:
Sorry for the delay. I have just removed the line.